Resources/doc/cookbook/recipe_dynamic_form_modification.rst

```diff
@@ -45,5 +45,7 @@ Then, you should be able to dynamically add needed fields to the form:
             if ($this->isCurrentRoute('create')) {
                 $formMapper->add('name', 'text');
             }
+
+            //$this->isCurrentRoute('edit', 'some.admin.code'); // is some.admin.code admin's edit page?
```

Comment:
Indeed, your example if more comprehensible.

@Rivsen Could you please update this part? Thanks.